src/native_background.ts

```diff
@@ -270,14 +275,34 @@ export async function getenv(variable: string) {
  You'll get both firefox binary (not necessarily an absolute path) and flags */
 export async function ffargs(): Promise<string[]> {
     // Using ' and + rather that ` because we don't want newlines
-    let output = await pyeval(
-        'handleMessage({"cmd": "run", ' +
-            '"command": "ps -p " + str(os.getppid()) + " -oargs="})["content"]',
-    )
-    return output.content.trim().split(" ")
+    if ((await browserBg.runtime.getPlatformInfo()).os === "win") {
+        return []
```

Comment:
This is somewhat misleading - shouldn't this throw an error?

Reply:
Fixed [here][0]. 

Currently `ffargs()` is unused for Windows specific things. I thought I'd need it to get `:restart` working. But eventually I didn't need it. 

If we need `ffargs()` or alike for other functionalities, I'll submit a different patch. Thanks.

[0]: https://github.com/gsbabil/tridactyl/commit/ef788a393f09c17858e10e42f61eb00f6a7ed96e#diff-9ddd9172e4a54cdd30d56e5f10d5facaR286